examples/flax/question-answering/utils_qa.py

```diff
@@ -147,9 +147,17 @@ def postprocess_qa_predictions(
                     # provided).
                     if token_is_max_context is not None and not token_is_max_context.get(str(start_index), False):
                         continue
+
+                    # We shouldn't get index errors now because of all the cehcks above, but for some reason, some
+                    # are still there, see https://github.com/huggingface/transformers/issues/15401#issuecomment-1024845936
+                    try:
+                        example_offsets = (offset_mapping[start_index][0], offset_mapping[end_index][1])
+                    except IndexError:
+                        continue
```

Comment:
Logging the error with the specific example would be quite nice here I think, to identify problematic examples

Reply:
I think I have the exact problem, just waiting from some additional info from the Datasets team and will push a better fix.